packages/gestalt/src/contexts/ExperimentProvider.js

```diff
@@ -20,5 +20,5 @@ export default ExperimentProvider;
 
 export function useExperimentContext(experimentName: string): Experiment {
   const experiments = useContext(ExperimentContext) ?? {};
-  return experiments[experimentName];
+  return experiments[experimentName] ?? {};
```

Comment:
This allows the nice destructuring usage, which would otherwise throw if the experiment isn't defined:
```js
const { anyEnabled: myExpEnabled } = useExperimentContext('my_exp');
```